src/lang/lwc_fr.properties

```diff
@@ -448,4 +448,20 @@ lwc.admin.dump.filenotfound=%dark_red%Failed to find the locale inside of the ja
 lwc.admin.dump.success=%dark_green%Dumped locale file to: %file%
 
 # Donation protection type
-donation=Donation
\ No newline at end of file
+donation=Donation
+
```

Comment:
On some locales you added protection.typechanged and for some you didn't (like here). Maybe you forgot? Would be good to remember to change all of them at the same time if you are going to add the keys.